nginx/fulltest-provider/README.md

```diff
@@ -32,6 +32,10 @@ Note that, the oqs-nginx-{LIBOQS_VERSION}.tgz package contains all required conf
 
 #### Activation
 
-Execute `/opt/nginx/sbin/nginx -c /opt/nginx/interop.conf` to start the test server.
+Execute `OPENSSL_CONF=/opt/openssl/.openssl/ssl/openssl.cnf /opt/nginx/sbin/nginx -c /opt/nginx/interop.conf` to start the test server.
```

Comment:
This doesn't look like an "under-the-hood" change invisible to users: Doesn't this warrant updating at least USAGE.md for `nginx`? Why is this apparently user-API changing version update necessary?